### PR TITLE
CDK: Updating the version matrix

### PR DESCRIPTION
**Pull request by kmurphypolygon**
https://docs.google.com/presentation/d/1tOE1nfA349nFDXZQImTl8K-0iI_z5DFpyl3tnBLqdeQ/edit?usp=sharing

### CODE REVIEW

File: docs/cdk/version-matrix.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The table below shows the version compatibility for CDK releases and related com
     <td>v0.2.0</td>
   </tr>
   <tr>
-    <td>Upgraded Validium</td>
+    <td>Upgraded validium</td>
     <td>6</td>
     <td>zkevm-node@0.4.2</td>
```
Comment (Sharonbc01) [inline docs/cdk/version-matrix.md]:
```suggestion
    <td>0.4.2</td>
```

Comment (gsilva5) [reply docs/cdk/version-matrix.md]: Let's remove the older version before fork6.  We know that we want to get everyone updated asap.

**Review by gsilva5 — APPROVED** [review]
Ask is to remove older versions and only keep the fork 6 and above for now.  And we will remove fork 6 once fork 8 is ready.  

Approving for now but let's keep this in mind.

File: docs/cdk/version-matrix.md [hunk 2]
```diff
@@ -22,42 +22,33 @@ The table below shows the version compatibility for CDK releases and related com
...
-    <td>zkevm-node@0.5.6 (Etrog+uLxLy)</td>
-    <td><a href="https://github.com/0xPolygon/cdk-validium-node/releases/tag/v0.5.6%2Bcdk">v0.5.6+CDK</a> </td>
+    <td><a href="https://github.com/0xPolygonHermez/zkevm-node/releases/tag/v0.5.13">v0.5.13 </a>(Etrog+uLxLy)</td>
+    <td><a href="https://hub.docker.com/layers/0xpolygon/cdk-validium-node/0.5.13-cdk.3/images/sha256-b83a0dbf954cc514fa129b8eff4d74e61a46cacbfbd70dce9780ee31d9b62770?context=explore">v0.5.13+cdk.3</a> </td>
     <td><a href="https://github.com/0xPolygon/cdk-data-availability/releases/tag/v0.0.5">v0.0.5</a></td>
-    <td><a href="https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v4.0.4">v4.0.4</a></td>
-    <td>Merged: <br>zkevm-contracts</td>
-    <td><a href="https://github.com/0xPolygonHermez/zkevm-bridge-service/releases/tag/v0.4.0">v0.4.0</a></td>
+    <td>Prover: <a href="https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v4.0.19">v4.0.19</a></td>
+    <td> <a href="https://github.com/0xPolygonHermez/zkevm-rom/tree/v4.0.0-fork.7">v4.0.0-fork.7</a></td>
```
Comment (Sharonbc01) [inline docs/cdk/version-matrix.md:41]: Please remove word fork 7 as the fork Id is in the second column.

File: docs/cdk/version-matrix.md [hunk 3]
```diff
@@ -22,42 +22,33 @@ The table below shows the version compatibility for CDK releases and related com
...
+    <td>Bridge: <a href="https://github.com/0xPolygonHermez/zkevm-bridge-service/releases/tag/v0.4.2">v0.4.2</a></td>
```
Comment (Sharonbc01) [inline docs/cdk/version-matrix.md:42]: Please remove the work Bridge as Bridge is the heading for that column

File: docs/cdk/version-matrix.md [hunk 4]
```diff
@@ -22,42 +22,33 @@ The table below shows the version compatibility for CDK releases and related com
...
     <td>7</td>
-    <td>zkevm-node@0.5.6 (Etrog+uLxLy)</td>
-    <td><a href="https://github.com/0xPolygon/cdk-validium-node/releases/tag/v0.5.6%2Bcdk">v0.5.6+CDK</a> </td>
+    <td><a href="https://github.com/0xPolygonHermez/zkevm-node/releases/tag/v0.5.13">v0.5.13 </a>(Etrog+uLxLy)</td>
+    <td><a href="https://hub.docker.com/layers/0xpolygon/cdk-validium-node/0.5.13-cdk.3/images/sha256-b83a0dbf954cc514fa129b8eff4d74e61a46cacbfbd70dce9780ee31d9b62770?context=explore">v0.5.13+cdk.3</a> </td>
     <td><a href="https://github.com/0xPolygon/cdk-data-availability/releases/tag/v0.0.5">v0.0.5</a></td>
-    <td><a href="https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v4.0.4">v4.0.4</a></td>
-    <td>Merged: <br>zkevm-contracts</td>
-    <td><a href="https://github.com/0xPolygonHermez/zkevm-bridge-service/releases/tag/v0.4.0">v0.4.0</a></td>
+    <td>Prover: <a href="https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v4.0.19">v4.0.19</a></td>
```
Comment (Sharonbc01) [inline docs/cdk/version-matrix.md:40]: Please remove the word Prover as the title of that column is ZK-EVM prover

**Pull request merged**